CLI.md

```diff
@@ -48,7 +48,7 @@ This document contains the help content for the `unleash-edge` command-line prog
   Default value: `3043`
 * `--instance-id <INSTANCE_ID>` — Instance id. Used for metrics reporting
 
-  Default value: `01HX9QXY3MNKKAMP7XRS6RG2Q4`
+  Default value: `01J20TW2327N1JHE5XYHBE55KM`
```

Comment:
Hmm won't this be a new ulid every time?
```suggestion
  Default value: Automatically generated ulid.
```

I guess the reason for this, and especially this change in the value, is that we just copy paste the CLI output 🙂

Reply:
Yup, good point. I'll update the default.

Reply:
Hmm, no I won't since this is fully auto generated. For now we'll just accept that this will update each time we generate CLI.md

Reply:
Hmm, I changed it to <GENERATED_ULID> now. We should see if we can have our generator replace that value automatically

Reply:
For now we'll have to remember to replace the value with the placeholder again if we regenerate CLI.md

Reply:
Yep, makes sense 👍 Either way just something I noticed that is out of scope for the PR, so no worries.